dev/com.ibm.tx.jta/resources/com/ibm/ws/Transaction/resources/TransactionMsgs.nlsprops

```diff
@@ -878,6 +878,11 @@ WTRN0154_ERROR_CHECKPOINT_NEW_TX=WTRN0154E: The server checkpoint request failed
 WTRN0154_ERROR_CHECKPOINT_NEW_TX.explanation=Liberty does not support applications that begin or require a transaction while the application starts when a server checkpoint is requested.
 WTRN0154_ERROR_CHECKPOINT_NEW_TX.useraction=Use the server checkpoint at=deployment option or modify the application to begin transactions after it has started.
 
+# 0 - Stack trace of call to create new tx
+WTRN0155_CHECKPOINT_NEW_TX_STACK=WTRN0155W: A new transaction was created during the server checkpoint request. Here is the stack trace of this thread when the transaction was created: {0}.
```

Comment:
Change the first sentence to "An application created a transaction during the..." 
Change the second sentence to "The {0} stack trace for the thread was captured when the transaction was created."